DatadogInternal/Tests/NetworkInstrumentation/HTTPHeadersWriterTests.swift

```diff
@@ -28,7 +28,7 @@ class HTTPHeadersWriterTests: XCTestCase {
     }
 
     func testWritingDroppedTraceContext_withAutoSamplingStrategy() {
```

Comment:
suggestion/ I think the `*AutoSamplingStrategy*` is a leftover after renaming `samplingStrategy` from `.auto` to `.headBased`. Could we update this name in all tests, so it is correct? 🏕️ 🧹

Reply:
sure